docs/tutorials/multimodal/advanced_topics/tensorrt.ipynb

```diff
@@ -0,0 +1,411 @@
+    "# Use first prediction for initialization (e.g., allocating memory)\n",
+    "y_pred = predictor.predict_proba(sample)\n",
+    "\n",
+    "pred_time = []\n",
+    "for _ in range(n_trails):\n",
+    "    tic = time.time()\n",
+    "    y_pred = predictor.predict_proba(sample)\n",
+    "    elapsed = time.time()-tic\n",
+    "    pred_time.append(elapsed)\n",
+    "    print(f\"elapsed (pytorch): {elapsed*1000:.1f} ms (batch_size={batch_size})\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "912bdb80",
+   "metadata": {},
+   "source": [
+    "## Prediction with TensorRT module\n",
+    "First, we need to export the module to ONNX, in order to use TensorrtExecutionProvider in onnxruntime for prediction."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0448f763",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_path = predictor.path\n",
+    "trt_predictor = MultiModalPredictor.load(path=model_path)\n",
+    "trt_predictor.optimize_for_inference()\n",
+    "\n",
+    "# Agagin, use first prediction for initialization (e.g., allocating memory)\n",
+    "y_pred_trt = trt_predictor.predict_proba(sample)\n",
+    "\n",
+    "clear_output()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e0bf01e0",
+   "metadata": {},
+   "source": [
+    "Under the hood, the `optimize_for_inference()` would generate an OnnxModule that can be a drop-in replacement of torch.nn.Module, and it replaced torch.nn.Module for optimized inference with onnxruntime."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "389f19cb",
+   "metadata": {},
+   "source": [
+    "Then, we can perform prediction or extract embeddings as usual. For fair inference speed comparison, here we run prediction multiple times."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7cb7de12",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pred_time_trt = []\n",
+    "for _ in range(n_trails):\n",
+    "    tic = time.time()\n",
+    "    y_pred_trt = trt_predictor.predict_proba(sample)\n",
+    "    elapsed = time.time()-tic\n",
+    "    pred_time_trt.append(elapsed)\n",
+    "    print(f\"elapsed (tensorrt): {elapsed*1000:.1f} ms (batch_size={batch_size})\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2462b437",
+   "metadata": {},
+   "source": [
+    "To verify the correctness of the prediction results, we can compare the results side-by-side.\n",
+    "\n",
+    "Let's take a peek at the expected results and TensorRT results."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bdbf5f57",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "y_pred, y_pred_trt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dcca6e00",
+   "metadata": {},
+   "source": [
+    "We can see the probabilities are quite close, and we should be able to safely assume these results are relatively close for most of the cases."
```

Comment:
Better to explain a little bit why there are some precision lost. A reference link of from nividia also helps.

Reply:
1) added explanation, 2) added reference link